en/px-panel-management.json

```diff
@@ -212,10 +212,6 @@
             "button": {
                 "addFilter": "Add filter"
             },
-            "panelists": "{{number}} panelists",
-            "panelistMatch": "**{{number}} panelist match** out of {{total}}",
-            "panelistsMatch": "**{{number}} panelists match** out of {{total}}",
-            "noPanelistsMatch": "**No panelist match** out of {{total}}",
```

Comment:
With the way translations get deployed at merge (I think?), do we need to leave these old ones in place until the change using them has been deployed to production? Or is this some currently unused bit of functionality?

Reply:
that is a good point, this is an internal admin tool though, so I'm willing to break some translations for a few minutes

Reply:
edit: going to keep the old tokens around for the moment, forgot it will be 2 weeks until we do a release, not 1